Google.Api.Generator.Tests/Invoker.cs

```diff
@@ -128,5 +134,10 @@ public static WithPath TempDir()
             Directory.CreateDirectory(dirPath);
             return new WithPath(() => Directory.Delete(dirPath, recursive: true), dirPath);
         }
+
+        private static (string ProtocFileKey, string RuntimeKey, string PluginFileKey) GetOSSpecificConfigs(string rootDir) =>
```

Comment:
You don't have to name the parts of the tuple, but if you want to do so, these are not keys anymore, they are the values so
```suggestion
        private static (string protocFile, string runtime, string pluginFileName) GetOSSpecificConfigs(string rootDir) =>
```